src/System.Private.Windows.Core/src/System/Value.DateTimeOffsetFlag.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System;
+
+internal readonly partial struct Value
+{
+    private sealed class DateTimeOffsetFlag : TypeFlag<DateTimeOffset>
+    {
+        public static DateTimeOffsetFlag Instance { get; } = new();
+
+        public override DateTimeOffset To(in Value value)
+            => new(new DateTime(value._union.Ticks, DateTimeKind.Utc));
```

Comment:
We had decided to break after `=>` if its needed https://github.com/dotnet/winforms/blob/main/docs/coding-style.md#line-breaks
```suggestion
        public override DateTimeOffset To(in Value value) =>
            new(new DateTime(value._union.Ticks, DateTimeKind.Utc));
```